CHANGELOG.md

```diff
@@ -87,4 +87,8 @@ Types of change:
 
 ### Fixed
 - [Python - Check Available Python Modules - Removed unnecessary `$` and general formatting improvements](https://github.com/enkidevs/curriculum/pull/2127)
-- [Git - What's Git? - Fixed spelling of Linus, creator of Linux](https://github.com/enkidevs/curriculum/pull/2129)
\ No newline at end of file
+- [Git - What's Git? - Fixed spelling of Linus, creator of Linux](https://github.com/enkidevs/curriculum/pull/2129)
+
+## June 1st 2020
+### Fixed
+- [Python - List Methods II - Fixed typo when using the #insert method ](https://github.com/enkidevs/curriculum/pull/2156)
```

Comment:
new entries should go at the top of the list so we can always check the newest changes/additions

Reply:
ah sorry my bad - have now corrected!